src/helpers/utilities.ts

```diff
@@ -31,3 +33,8 @@ export const loadScript = (src: string): Promise<void> =>
     script.onload = () => resolve();
     return (document.body && document.body.appendChild(script)) || reject();
   });
+
+export const isMobile = () => {
+  if (isBrowser) return window.innerWidth <= 768;
+  return false;
+};
```

Comment:
```suggestion
export const isMobile = isBrowser && window.innerWidth <= 768;
};
```
Was thinking maybe this doesn't need to be a function? WDYT? :)